### PR TITLE
qiaohai.py first submission

### PR DESCRIPTION
**Pull request by joeseaer**
@dragonyanglong, thanks for your coding tutoria, this PR is for me to learn github. please review and merge.

### CODE REVIEW

File: people/qiaohai.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import numpy.matlib
```
Comment (dragonyanglong) [inline people/qiaohai.py]: please remove this line, which is not needed for running the code here.

File: people/qiaohai.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import numpy.matlib 
+import numpy as np 
+
+# Generate a diagonal matrix with 3 rows and 4 columns.  
+print (np.matlib.eye(n =  3, M =  4, k =  0, dtype =  float))
```
Comment (dragonyanglong) [inline people/qiaohai.py]: code style suggestions: please refer to PEP-8 coding style and update this line of code. (basically, we just need one space before and after "=".

**Review by dragonyanglong — COMMENTED** [review]
Hi @joeseaer , nice job for learning the git workflow quickly. I left some quick comments for your codes. You can push updated codes to your branch (the same branch you created the PR), and git will automatically show your updates here on the website. Then @ me to review the PR and merge. 

BTW, you can try writing the git commit messages using numpy doc style (as I suggested in the tutorial), such as: `ENH: first submission to learn github`. It uses the prefix three letters to distinguish what your work is.

Overall, very good!

**Comment by dragonyanglong** [comment]
After you resolve the two comments I left, please click "resolve conversation", then @ me, so it reflects you solved all the issues.

**Comment by joeseaer** [comment]
Hi @dragonyanglong, I'm sorry for the careless submission yesterday. I modified the code after referring to the PEP8 and the numpy doc. Please review the PR, Thanks!

**Pull request merged**